macOS/DuckDuckGo/Permissions/Model/PermissionModel.swift

```diff
@@ -172,7 +172,7 @@ final class PermissionModel {
                 .systemAuthorizationDenied(systemWide: !geolocationService.locationServicesEnabled())
         }
 
-        permissions.forEach { self.permissions[$0].authorizationQueried(query) }
+        permissions.forEach { self.permissions[$0].authorizationQueried(query, updatesRequest: $0 == .popups) }
```

Comment:
I'm using a safe and simple approach here because I want to avoid breaking other stuff.

That said, I have the impression this might be a good default for all authorization queries.